ui/mirage/factories/job.js

```diff
@@ -18,14 +21,11 @@ export default Factory.extend({
   groupsCount: () => faker.random.number({ min: 1, max: 2 }),
 
   region: () => 'global',
-  type: faker.list.random(...JOB_TYPES),
+  type: faker.helpers.randomize(JOB_TYPES),
   priority: () => faker.random.number(100),
   all_at_once: faker.random.boolean,
-  status: faker.list.random(...JOB_STATUSES),
-  datacenters: provider(
-    () => faker.random.number({ min: 1, max: 4 }),
-    faker.list.random(...DATACENTERS)
-  ),
+  status: faker.helpers.randomize(JOB_STATUSES),
+  datacenters: faker.helpers.randomize(DATACENTERS, faker.random.number({ min: 1, max: 4 })),
```

Comment:
This doesn't look quite like the right translation. 

If I understand `faker.helpers.randomize` correctly, it always returns a random element of an array, but we want `datacenters` to be a random lengthed array of randomly chosen datacenters.

I think this is the right translation:

```js
datacenters: provider(
  () => faker.random.number({ min: 1, max: 4 }),
  faker.helpers.randomize(DATACENTERS))
),
```

It'd be nice to get rid of my scrappy utils though!

Reply:
eep, you’re right, it’s a mistake left over from when I was using `faker.randomize.arrayElements`, which would be nice to have, but isn’t in a released version.

That implementation seems a bit off to me as it could result in there being duplicate datacentres, so I used `faker.helpers.randomize` and a randomised index for `slice`. Thanks for catching this!